debian/patches/wazo_stun_recurring_resolution

```diff
@@ -312,13 +321,17 @@ Index: asterisk-18.8.0/res/res_rtp_asterisk.c
 +	}
 +
 +	ao2_rdlock((struct stun_resolver *) stun);
++	if (!stun->address) {
++		ao2_unlock((struct stun_resolver *) stun);
```

Comment:
I don't think you need to cast `stun` here.

`grep` tells me that `ao2_rdlock(stun);` and `ao2_unlock(stun);` is usually used in the Asterisk code base.

Reply:
The casts are to prevent ao2_unlock's warning on stun variable being `const struct stun_resolver *`.  The const should probably be removed from the function signature's

Reply:
+1 for removing the const from `stun_resolver_get_resolved` signature